lib/parser.pegjs

```diff
@@ -205,11 +222,24 @@ lit
     { throw new peg$SyntaxError("Illegal TeX function", [], f, location()); }
 
 litstuff = //basically lit without DELIMITER or LITERAL as an option (replaced by LITERALPART at the end), needed because they interrupt the parsing of commands in PAREN.
-    f:PAREN atsymbol atsymbol l:lit      { return ast.Tex.PAREN2(f, l); }
-  / f:PAREN atsymbol l:lit { return ast.Tex.PAREN1(f, l); }
-  / l1:litparen1 "_" l2:lit     { return ast.Tex.POCH(l1, l2); }
-  / f:PAREN l:litparen1  { return ast.Tex.PAREN1(f, l); } //must be before LITERAL!!
-  / name:PAREN l:lit            { return ast.Tex.PAREN2(name, l); }
+    f:PAREN atsymbol atsymbol l:lit   &{ return options.semanticLaTeX; }
+    { return ast.Tex.PAREN2(f, l); }
+  / f:PAREN atsymbol l:lit &{ return options.semanticLaTeX; }
+    { return ast.Tex.PAREN1(f, l); }
+  / "P_" l:lit "^" "{"* _ PAREN_OPEN _ l2:lit _ "," _ l3:lit _ PAREN_CLOSE _ "}"* l4:litparen1 &{ return options.semanticLaTeX; }
```

Comment:
This part of the code (and just below) does the replacements, and needs to be moved into the texer repository. Furthermore, the texer repository needs to ultimately get this information from the csv file.